atlp-rwanda/e-commerce-bitcrafters-bn
# [#187364752] Feature: Make Payment using Stripe API

**What does this PR do?**

This pull request adds a new feature to process payments using Stripe. It includes the ability to handle payments for orders and update the order status once the payment is successful.

 **Description of Task to be completed?**

**Process:**

- The system checks if the order exists and if it belongs to the user.
- Creates a payment intent with Stripe and confirms it.
- Updates the order status to "COMPLETED" if the payment is successful.

**Response:**

Returns a success message and order confirmation details.

 **How should this be manually tested?**

- Use Postman or a similar tool to send a payment request.
- Check if the order status updates correctly in the database.


 **What are the relevant pivotal tracker/Trello stories?**
#187364752

 Screenshots or video (if appropriate)
https://www.loom.com/share/abafc20c3d5f4388bca143a98eb337a2?sid=72c6cdba-19d9-4d4f-b8ee-a65773df10e9
